closure/goog/events/pastehandler.js

```diff
@@ -94,6 +94,7 @@ goog.events.PasteHandler = function(element) {
 
   if (goog.userAgent.WEBKIT ||
```

Comment:
I guess it depends what Google's minimum supported versions are?

I'd happily drop it... :)